qiita_pet/templates/redbiom.html

```diff
@@ -0,0 +1,175 @@
+{% from qiita_core.qiita_settings import qiita_config %}
+{% from future.utils import viewitems %}
+{% extends sitebase.html %}
+
+{%block head%}
+<link rel="stylesheet" href="{% raw qiita_config.portal_dir %}/static/vendor/css/select2.min.css" type="text/css">
```

Comment:
Just checked - `select2.min.js` it is added in `sitebase.html` but the css is not. I would suggest removing them from here and adding the css in the `sitebase.html` file. Since the JS is added there, somewhere in the code it is expected to be available but currently is not, which I can generate some problems down the line.

On the other hand, `sharing.js` is ok to leave here since it is only used here.